core/arch/arm/include/sm/watchdog_smc.h

```diff
@@ -0,0 +1,93 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) STMicroelectronics 2023
+ */
+#ifndef WATCHDOG_SMC_H
+#define WATCHDOG_SMC_H
```

Comment:
`SM_WATCHDOG_SMC_H`

Reply:
I've made same as optee_smc.h which use OPTEE_SMC_H.
Only SM and PM use this, I'm ok but difficult to get consistency with other file.
No issue to put SM whatever, not seeing an explicit rule to use the folder name (which is not used in the other include folders)

Reply:
_optee_smc.h_ header file is there since a while, before OP-TEE tried to harmonize the header file guards.
I think _optee_smc.h_  could be updated accordingly. I guess some other existing legacy header files also.